study.md

```diff
@@ -21,6 +21,11 @@ Ember](https://www.codeschool.com/courses/try-ember) course from CodeSchool.
 
 In your own words, describe why someone would use Ember.
 
-```md
-<!-- your answer here -->
-```
+You would use Ember.js framework to build a fast-loading interactive single-page web app. The framework is fairly easy to get started in. A lot of the Model-View-Controller files and handlebars templates are generated for you via command line scripts much like Rails.
```

Comment:
Yes, you are right that ember is a framework like rails, and you can generate an element from the command line like rails.